megamek/src/megamek/codeUtilities/MathUtility.java

```diff
@@ -133,8 +133,28 @@ public static float clamp(final float value, final float min, final float max) {
      *         clamp(7, 6, 8) returns 7,
      *         clamp(12, 3, 5) returns 5.
      */
-    public static long clamp(final long value, final long min, final long max) {
+    public static long clamp(long value, long min, long max) {
         return Math.min(Math.max(value, min), max);
     }
     // endregion Clamp
+
+    // region Clamp01
+    /**
+     * Clamp a value between 0 and 1.0 (inclusive).
+     * @param value the float value to clamp
+     * @return The value clamped
+     */
+    public static float clamp01(float value) {
+        return Math.min(Math.max(value, 0f), 1f);
+    }
+
+    /**
+     * Clamp a value between 0 and 1.0 (inclusive).
+     * @param value the float value to clamp
+     * @return The value clamped
+     */
+    public static double clamp01(double value) {
+        return Math.min(Math.max(value, 0f), 1f);
+    }
```

Comment:
Although meant to be default clamp ranges, why not just be helper methods for the regular `clamp` methods and pass in the default 0 and 1 values?

Reply:
its minimalistic enough that it makes no difference.